apps/emqx_machine/README.md

```diff
@@ -1,5 +1,48 @@
-# EMQX Machine
+# emqx_machine
```

Comment:
Q: should this be the slug for the application?  I had understood that it should be a "pretty" version of the name :thinking:

Reply:
Thanks, fixed.